### ISSUE TITLE
Categories pages doesn't load from the NMT but loads from the PC (r2595)

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#2051]
Original [issue 2052](https://code.google.com/p/moviejukebox/issues/detail?id=2052) created by Omertron on 2011-07-27T13:03:10.000Z:

Hi!

As mentionned in a previous issue (which was closed, I don't know why), the number of actors indexed has suddenly increased (I don't know which release is concerned, but there were more information in the issue that was closed and that I can't find any more).
With 2,700 movies, my jukebox size was 5Gb
With 2,900 movies, it jumped to 18Gb, without any changes in my configuration files.
In other words, when the movies count is increased by 7%, the library's size is increased by 160%!

This is the first issue - I wish you don't close it without having a look... Thanks.

Second issue: Now the categories page doesn't load any more from my C-200. It hangs and I have to reboot. But if I call the categories page from my PC, it is correctly displayed (I have saved the full html page on my PC - attached to this issue).

Thanks for your help!
